components/bl-list-component/src/index.js

```diff
@@ -0,0 +1,18 @@
+import { useMemo } from 'react'
+
+import { List } from './list'
+
+const { cn } = BackendlessUI.CSSUtils
+
+export default function ListComponent ({ component, eventHandlers }) {
+
+  const { style, classList, width, backgroundColor } = component
+
+  const styles = useMemo(() => ({ style, width, backgroundColor }), [style, width, backgroundColor])
```

Comment:
const styles = { ...style, width, backgroundColor }. Also we could do it props without creating variable.

There are no need in useMemo as we haven't any calculations